lib/db/mysql.js

```diff
@@ -199,14 +199,16 @@ module.exports = function (log, error) {
         sessionToken.uaBrowserVersion,
         sessionToken.uaOS,
         sessionToken.uaOSVersion,
-        sessionToken.uaDeviceType
+        sessionToken.uaDeviceType,
+        sessionToken.tokenVerificationId
```

Comment:
What will happen if the auth-server somehow forgets to pass in a `tokenVerificationId` here.  Will it error, or insert with an empty string?  (AFAICT we don't really validate the incoming data to any of the endpoints in this serverl, but it's worth thinking about whether we should start doing so).